### ISSUE TITLE
Add CITATION.cff

### ISSUE THREAD

**Issue by hshuaib90** [Project-MONAI/monai-deploy#59]
I think we need a CITATION.cff file like the core MONAI repo so that people can reference the repository appropriately in written publications. Eventually I hope there will be a journal paper that people can reference instead.

**Comment by dbericat** [comment]
Agreed, want to do it and submit the PR? I will review and approve.
